crates/router/src/connector/stax/transformers.rs

```diff
@@ -11,6 +11,37 @@ use crate::{
     types::{self, api, storage::enums},
 };
 
+#[derive(Debug, Serialize)]
+pub struct StaxRouterData<T> {
+    pub amount: i64,
```

Comment:
Let amount be f64
```suggestion
    pub amount: f64,
```

Reply:
Ohh thank you for pointing out my changes